gammapy/utils/deprecation.py

```diff
@@ -0,0 +1,54 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+__all__ = [
+    "GammapyDeprecationWarning",
+    "deprecated",
+    "deprecated_renamed_argument",
+    "deprecated_attribute",
+]
+
+
+class GammapyDeprecationWarning(Warning):
+    """
+    The Gammapy deprecation warning
+    """
+
+
+def deprecated(since, **kwargs):
+    """
+    Used to mark a function or class as deprecated.
+
+    Reuses Astropy's deprecated decorator.
+    Check arguments and usage in `~astropy.utils.decorator.deprecated`
+
+    Parameters
+    ----------
+    since : str
+        The release at which this API became deprecated.  This is required.
+    """
+    from astropy.utils import deprecated
```

Comment:
Any particular reason why the imports are inline in the function and not at the top?

Reply:
Mostly to avoid confusion with the function names. Maybe importing `astropy.utils` and calling from the namespace would be better.